paddle/fluid/pir/dialect/op_generator/op_interface_gen.py

```diff
@@ -149,7 +149,7 @@ def gen_op_vjp_str(
             index_0 = fwd_outputs_list.index(bw_input_name)
         else:
             vjp_param_name = 'out_grads'
-            grad_idx += 1
+            grad_idx = fwd_outputs_list.index(bw_input_name[:-5])
```

Comment:
```suggestion
           offset = len("_grad")
            grad_idx = fwd_outputs_list.index(bw_input_name[:-5])
```
这里建议不要直接写 -5，不熟悉背景的其他开发者会比较奇怪这个magic_number 是怎么来的。之前我升级gen.py时遇到过影响其他magic_number导致CI失败问题，排查起来不是很友好。

Reply:
好的，后面fix一下